_projects/ia32-generic-qemu/lwip/lwipopts.h

```diff
@@ -0,0 +1,91 @@
+#define LWIP_TCPIP_CORE_LOCKING 1
+#define LWIP_SUPPORT_CUSTOM_PBUF 1
+#define LWIP_NETIF_LOOPBACK 1
+#define LWIP_HAVE_SLIPIF 0
+#define LWIP_NETIF_API 1
+#define LWIP_SOCKET 1
+#define LWIP_COMPAT_SOCKETS 0
+#define LWIP_ARP 1
+#define LWIP_ICMP 1
+#define LWIP_RAW 1
+#define LWIP_DHCP 1
+#define LWIP_DNS 1
+#define LWIP_AUTOIP 1
+#define LWIP_UDP 1
+#define LWIP_TCP 1
+#define LWIP_TCP_KEEPALIVE 1
+#define MEM_LIBC_MALLOC 1
+#define MEMP_MEM_MALLOC 1
+#define LWIP_ERRNO_INCLUDE "errno.h"
+#define LWIP_DNS_API_DEFINE_ERRORS 0
+#define LWIP_DNS_API_DEFINE_FLAGS 0
+#define LWIP_DNS_API_DECLARE_STRUCTS 0
+#define LWIP_DNS_API_DECLARE_H_ERRNO 0
+#define MEMP_NUM_NETCONN 1024
+#define PPP_SUPPORT 1
+#define PPPOS_SUPPORT 1
+#define PAP_SUPPORT 1
+#define CHAP_SUPPORT 1
+#define MSCHAP_SUPPORT 1
+#define LWIP_TIMEVAL_PRIVATE 0
+#define LWIP_IPV6 1
```

Comment:
- maybe move it to separate section so the addition of ipv6 (over default config would be more visible
- consider defining larger `LWIP_IFSTATUS_DEV_BUFFER_SIZE` if you feel the need (multiple ipv6 addresses?